.vscode/settings.json

```diff
@@ -11,6 +11,6 @@
     "editor.defaultFormatter": "dprint.dprint"
   },
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
whats this do

Reply:
I think the vscode eslint extension just wants to change its config, it keeps automatically changing and I'm tired of having it only in my local.